Makefile

```diff
@@ -101,6 +101,10 @@ release-test:
 release-all:
 	mkdir -p $(builddir)/release
 	cd $(builddir)/release && cmake -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=release $(topdir) && $(MAKE)
+	
+release-rpi:
+	mkdir -p $(builddir)/release-rpi
+	cd $(builddir)/release-rpi && cmake -D NO_AES=ON -D BUILD_TESTS=OFF -D CMAKE_BUILD_TYPE=Release $(topdir) && $(MAKE)
```

Comment:
`BUILD_TESTS` is `OFF` by default. I see no reason to disable it explicitly, especially since the default `release` target does it neither.